src/currency.rs

```diff
@@ -13,7 +14,7 @@ pub struct Currency {
 
     // The estimated millisatoshis per smallest "unit" of this currency (eg. 1 cent in USD).
     #[serde(rename = "multiplier")]
-    pub millisatoshi_per_unit: i64,
+    pub millisatoshi_per_unit: f64,
```

Comment:
I think you need to remove the `Eq` for this struct. Rust does not implement Eq for float values.

Reply:
Yeah :-/ I guess I'd also need to remove eq on anything that uses this then. I can do that though, it seems.